### DISCUSSION TITLE
CustomWebView make the companion crashed

### DISCUSSION THREAD

**Discussion by samskomsel** [vknow360/CustomWebView#13] [Q&A] [answered]
Hi, i am making simple block to test this CustomWebView
![Capture](https://user-images.githubusercontent.com/87565734/126033153-0e370875-c687-4b7b-a227-00c3037c7116.PNG)
but when I try it on companion, companion crashes immidiately. I have tried to do the same in standard WebView but it doesn't crash

**Comment by vknow360** [comment, marked as answer]
I would like to point out two things.
1) Since CustomWebView is an extension so it can't create visible components directly.
You need to create them yourself.
Refer to this guide:
https://community.kodular.io/t/how-to-use-customwebview-extension/78732

2) Probably WebViewer can accept an url without protocol but not CustomWebView.
You need to prefix `http://` or `https://` before url.

**Reply by samskomsel** [reply]
Thanks
